src/web/components/MatchNav.tsx

```diff
@@ -204,23 +211,101 @@ const Border = () => (
     />
 );
 
-export const MatchNav = ({ homeTeam, awayTeam, comments }: Props) => (
-    <StretchBackground>
-        <Row>
-            <TeamNav
-                name={homeTeam.name}
-                score={homeTeam.score}
-                crest={homeTeam.crest}
-                scorers={homeTeam.scorers}
-            />
-            <Border />
-            <TeamNav
-                name={awayTeam.name}
-                score={awayTeam.score}
-                crest={awayTeam.crest}
-                scorers={awayTeam.scorers}
-            />
-        </Row>
-        {comments && <Comments comments={comments} />}
-    </StretchBackground>
+const thinGreySolid = `1px solid ${border.secondary}`;
+
+const GreyBorder = () => (
+    <div
+        className={css`
+            /* stylelint-disable-next-line color-no-hex */
+            border-left: ${thinGreySolid};
+            margin-left: ${space[1]}px;
+            width: ${space[2]}px;
+        `}
+    />
+);
+
+const tabsContainer = css`
+    display: flex;
+    position: relative;
+    border-bottom: ${thinGreySolid};
+
+    li {
+        flex-basis: 50%;
+        height: 40px;
+        border-top: 3px solid ${border.secondary};
+
+        :nth-child(1) {
+            border-top: 3px solid ${palette.sport.main};
+        }
+        :nth-child(2) {
+            border-left: ${thinGreySolid};
+        }
+
+        span {
+            ${headline.xxxsmall()};
+            background: transparent;
+            padding: 6px 8px 0;
+            text-align: left;
+            font-weight: 600;
+            min-height: 36px;
+            display: block;
+            width: 100%;
+        }
+        a span:hover {
+            background-color: ${palette.neutral[93]};
+        }
+
+        a span {
```

Comment:
This is a little brittle as you are overriding spans based on the parent without context of the previous styling.

It would be more idomatic of DCR styling to add `className`'s for 'tab', `tabLabel` and `tabAnchor` and style off of these. You can then also style within `tabLabel` like 

```
// tabLabel is the `span`
tabLabel = css`
& a {
	// styles that only apply inside an anchor
}

& a:hover {
	// styles that only apply inside a hovered anchor
}
`
```

this will make it easier to reason with the code and contextualise the styling to the parent

Reply:
@gtrufitt could you take another look? Happy to go through it on a call if needs be.